openedx/features/lti_course_tab/tab.py

```diff
@@ -34,7 +34,38 @@ class LtiCourseLaunchMixin:
     }
     DEFAULT_ROLE = 'Student'
 
+    def _get_pii_lti_parameters(self, course: CourseBlock, request: HttpRequest) -> Dict[str, str]:
+        """
+        Get LTI parameters that contain PII.
+
+        Args:
+            course (CourseBlock): CourseBlock object.
+            request (HttpRequest): Request object for view in which LTI will be embedded.
+
+        Returns:
+            Dictionary with LTI parameters containing PII.
+        """
+        lti_config = self._get_lti_config(course)
```

Comment:
We can make the calls below a bit less duplicative by memo-izing `lti_config.lti_config`, since we only ever need the internal variable anyway..

```suggestion
        lti_config = self._get_lti_config(course).lti_config
```